uber/templates/emails/test_email.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base-admin.html" %}
+{% block title %}Test email sending{% endblock %}
+{% block content %}
+
+<h2>Send a test email from RAMS. Please use with care.</h2>
+
+<form action="test_email" method="POST">
+    <table border="0">
+        <tr>
+            <td>To:</td>
+            <td><input size="50" type="text" name="to_address" value="{{to_address}}" /></td>
+        </tr>
+        <tr>
+            <td>From:</td>
+            <td><input size="50" type="text" name="from_address" value="{{from_address}}" /></td>
+        </tr>
+        <tr>
+            <td>Subject:</td>
+            <td><input size="50" type="text" name="subject" value="{{subject}}" /><br/></td>
+        </tr>
+    </table>
+
+    Body:<br/>
+    <textarea cols=100 rows=20 name="body">{{body}}</textarea>
+
+    <br/><br/>
+    <input type="submit" value="submit" name="submit">
```

Comment:
If you remove the ``value="submit"`` from this field, it won't pass a parameter for the submit button, and we can get rid of the ``**params``.